tests/e2e/contracts/log_indexer.py

```
@@ -0,0 +1,68 @@
+# {
+#   "Seq": [
+#     { "Depends": "py-lib-genlayer-embeddings:test" },
+#     { "Depends": "py-genlayer:test" }
+#   ]
+# }
+
+import numpy as np
+from genlayer import *
```

Ruff (0.11.9)

`from genlayer import *` used; unable to detect undefined names

(F403)